src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_NUMBER = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int RANDOM_SIDE = 10;
+    private ColorSupplier supplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {

```

Comment:
There is a typo in the method name `getRundomFigure()`. It should be corrected to `getRandomFigure()` to ensure consistency and avoid confusion.